src/model/press-package.v1.yaml

```diff
@@ -0,0 +1,28 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Press package
+type: object
+allOf:
+  - $ref: ../snippets/press-package.v1.yaml
+  - properties:
+        content:
+            description: Content
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        relatedContent:
+            title: Related content
+            type: array
+            items:
+                oneOf:
+                  - $ref: ../snippets/article-poa.v1.yaml
+                  - $ref: ../snippets/article-vor.v1.yaml
+            minItems: 1
+            uniqueItems: true
+        mediaContacts:
+            title: Media contacts
+            type: array
+            items:
+                $ref: ../misc/person.v1.yaml
```

Comment:
@thewilkybarkid We need to allow an array of people with the option properties of email addresses and phone numbers. Please can you advise on the best approach for this.